steno/src/model/stream/stream.cpp

```diff
@@ -101,11 +103,11 @@ void Stream::stop()
 {
     updateState(IStream::State::Stopping);
 
-    if (m_odasClient->getState() != OdasClientState::CRASHED)
-    {
-        m_odasClient->stop();
-        m_odasClient->join();
-    }
+     if (m_odasClient->getState() != OdasClientState::CRASHED)
+     {
+         m_odasClient->stop();
+         m_odasClient->join();
+     }
```

Comment:
format error?

Reply:
should travis fix it?

Reply:
travis no longer checks formatting because it's broken.